### PR TITLE
CVM - Cardano Version Manager :)

### PR DESCRIPTION
**Pull request by orelvis15**
## Showcase addition

* Name: CVM
* Description: Version manager for cardano node
* Link:   [Project](https://github.com/orelvis15/cvm)
* Source: [Source Code](https://github.com/orelvis15/cvm)
* Tags:
  * `opensource`
  * `operatortool `

### CODE REVIEW

**Review by katomm — CHANGES_REQUESTED** [review]
Thank you very much. Looks more like a builder tool to me, opinions on this? My suggestion is also to remove the "ecosystem" tool, it is for websites that map out the ecosystem like https://developers.cardano.org/showcase/?tags=ecosystem

**Comment by katomm** [comment]
Any update on this @orelvis15?

**Comment by orelvis15** [comment]
> Any update on this @orelvis15?

is anything else needed for the merge?

**Comment by orelvis15** [comment]
> Any update on this @orelvis15?

I'm ready!, there I uploaded the changes.

**Review by rphair — CHANGES_REQUESTED** [review]
@orelvis15 the tag you have applied (`pooltool`) puts it into this category of the Showcase for sites showing stake pool data for delegation & analytics purposes, like these: https://developers.cardano.org/showcase/?tags=pooltool

Your tool is for pool _operators_ and should therefore be tagged as @katomm says as a Builder tool but with the `operatortool` tag, like these: https://developers.cardano.org/tools?tags=operatortool

Finally if this is all true then the thumbnail also needs to be moved into the corresponding directory.  By the way, your tool looks great 🤓

File: src/data/showcases.js [hunk 1]
```diff
@@ -1049,6 +1049,15 @@ const Showcases = [
     source: null,
     tags: [ "analytics", "explorer"],
   },
+  {
+    title: "CVM",
+    description:
+      "Cardano Version Manager, manage the configuration and versions of your pool.",
+    preview: require("./showcase/cvm.png"),
```
[the remark this answers is not in the document]
Comment (orelvis15) [reply src/data/showcases.js]: Done!

File: src/data/showcases.js [hunk 2]
```diff
@@ -1049,6 +1049,15 @@ const Showcases = [
...
+    website: "https://sites.google.com/view/cvmcli/inicio",
+    source:  "https://github.com/orelvis15/cvm",
+    tags: ["pooltool"],
```
[the remark this answers is not in the document]
Comment (orelvis15) [reply src/data/showcases.js]: Ho, sorry I hadn't noticed! I already left the change ready. @rphair  thanks a lot!

**Review by rphair — APPROVED** [review]
looks good now 👍

**Review by katomm — APPROVED** [review]

**Pull request merged**